x-pack/plugins/serverless_search/public/layout/nav.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CoreStart } from '@kbn/core/public';
+import {
+  ChromeNavigationNodeViewModel,
+  Navigation,
+  NavigationKibanaProvider,
+} from '@kbn/shared-ux-chrome-navigation';
+import React from 'react';
+
+const navItems: ChromeNavigationNodeViewModel[] = [
+  {
+    title: '',
+    id: 'root',
+    items: [
+      { id: 'overview', title: 'Overview', href: '/app/enterprise_search/overview' },
+      { id: 'indices', title: 'Indices', href: '/app/enterprise_search/content/search_indices' },
+      { id: 'engines', title: 'Engines', href: '/app/enterprise_search/content/engines' },
+      { id: 'api_keys', title: 'API keys', href: '/app/management/security/api_keys' },
+      {
+        id: 'ingest_pipelines',
+        title: 'Ingest pipelines',
+        href: '/app/management/ingest/ingest_pipelines',
+      },
+    ],
+  },
+];
+
+export const createServerlessSearchSideNavComponent = (core: CoreStart) => () => {
```

Comment:
We are inside the `serverless_search` plugin. I think we can simply call this `createSideNavComponent`